pages/dkp/kommander/2.1/workspaces/applications/catalog-applications/zookeeper-operator/index.md

```diff
@@ -0,0 +1,59 @@
+---
+layout: layout.pug
+navigationTitle: ZooKeeper Operator
+title: ZooKeeper Operator
+menuWeight: 20
+excerpt: Information about the ZooKeeper Operator
+---
+
+## Overview
+
+[ZooKeeper](https://zookeeper.apache.org/index.html) is a centralized service for maintaining configuration information, naming, providing distributed synchronization, and providing group services. The [ZooKeeper operator](https://github.com/pravega/zookeeper-operator) is a Kubernetes operator that handles the provisioning and management of ZooKeeper clusters. It works by watching custom resources, such as `ZookeeperClusters`, to provision the underlying Kubernetes resources (`StatefulSets`) required for a production-ready ZooKeeper Cluster.
+
+## Installation
+
+1.  Follow the generic installation instructions for workspace catalog applications on the [Application Deployment](../application-deployment/) page.
+
+1.  Within the `AppDeployment`, update the `appRef` to specify the correct `zookeeper-operator` App. You can find the `appRef.name` by listing the available `Apps` in the workspace namespace:
+
+    ```bash
+    kubectl get apps -n ${WORKSPACE_NAMESPACE}
+    ```
+
+For details on custom configuration for the operator, please refer to the [ZooKeeper operator Helm Chart documentation](https://github.com/pravega/zookeeper-operator/tree/master/charts/zookeeper-operator#configuration).
+
+## Uninstall via the CLI
+
+Uninstalling the ZooKeeper operator will not directly affect any running `ZookeeperCluster`s. By default, the operator will wait for any `ZookeeperClusters` to be deleted before it will fully uninstall (you can set `hooks.delete: true` in the application configuration to disable this behavior). After uninstalling the operator, you will need to manually clean up any leftover Custom Resource Definitions (CRDs).
+
+1.  View ZookeeperClusters in all namespaces:
+
+    ```bash
+    kubectl get zookeeperclusters -A
+    ```
+
+1.  Deleting a `ZookeeperCluster`:
+
+    ```bash
+    kubectl delete zookeepercluster <name of zookeepercluster>
+    ```
```

Comment:
Should we mention this in the CR docs and then provide link to CR docs here?

Reply:
yeah we definitely can, I wasn't sure how much detail about uninstalling should go here. maybe the  `View ZookeeperClusters in all namespaces:` should also go in the CR docs

Reply:
from our slack convo, we decided that the CR docs should go in this PR for completeness so I added those in  0e0547a123ef471771aaddce5476347569f13ed6 (along with this suggestion ^)

Reply:
I think anything related to CR can go to CR docs and we just do a link here, commands about CRDs though should stay.

Reply:
yup agreed, fixed in 0e0547a123ef471771aaddce5476347569f13ed6